csrankings-b.csv

```diff
@@ -764,7 +764,7 @@ Brian Keegan,University of Colorado Boulder,http://www.colorado.edu/cmci/people/
 Brian Kocoloski,Washington University in St. Louis,https://www.cse.wustl.edu/~brian.kocoloski,Qu_xQjYAAAAJ
 Brian Kulis,Boston University,http://people.bu.edu/bkulis,okcbLqoAAAAJ
 Brian L. Curless,University of Washington,http://homes.cs.washington.edu/~curless,tlh8i7gAAAAJ
-Brian Logan,University of Aberdeen,https://www.abdn.ac.uk/people/brian.logan,tnSWU68AAAAJ
+Brian Logan 0001,University of Aberdeen,https://www.abdn.ac.uk/people/brian.logan,tnSWU68AAAAJ

```

Comment:
Actually needs two entries (one for each alias listed here: https://dblp.org/pid/81/6813.html).